src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/VisitRepository.kt

```diff
@@ -35,6 +35,14 @@ interface VisitRepository : JpaRepository<Visit, Long>, JpaSpecificationExecutor
   )
   fun findByReference(reference: String): Visit?
 
+  @Query(
+    "SELECT * FROM visit " +
```

Comment:
I think this query will also return the SUPERSEDED_CANCELLATION records. The query might need to be updated to excluded them.

Reply:
SUPERSEDED_CANCELLATION is the outcome status

Reply:
As discussed, yes we need to show these visits.